src/rgw/rgw_op.h

```diff
@@ -36,10 +36,124 @@
 #include "include/assert.h"
 
 using namespace std;
+using ceph::crypto::SHA1;
 
 struct req_state;
 class RGWHandler;
 
+/* get torrent */
+#define INFO     dout(1) <<__FILE__<<":"<<__func__<<":"<<__LINE__<<":"<<
```

Comment:
please take this macro out and use `ldout(s->cct, #)` directly. the file/func/line are unnecessary